submissions/tavor118sn/task_friends_app/script.js

```diff
@@ -0,0 +1,156 @@
+// Global variables
+
+const CARD_LIST = document.getElementById('cardList');
+const FORM = document.forms.mainForm;
+const ASCENDING_ORDER = 'ascending';
+const DESCENDING_ORDER = 'descending';
+
+let friends;
+
+// URL
+
+const USER_FIELDS = ['gender', 'name', 'email', 'picture', 'dob'];
+
+const SITE_URL = 'https://randomuser.me/api/';
+const BASE_URL = new URL(SITE_URL);
+BASE_URL.searchParams.set('nat', 'us');
+BASE_URL.searchParams.set('results', '12');
+BASE_URL.searchParams.set('inc', USER_FIELDS.join());
+
+const buildUrl = (queryParam, paramValue) => {
+  const newUrl = new URL(BASE_URL);
+  if (queryParam && paramValue) {
+    newUrl.searchParams.set(queryParam, paramValue);
+  }
+  return newUrl;
+};
```

Comment:
TBH, I am not completely following why this logic outside function which makes a request?

Reply:
Moved to `loadUserList`